### PR TITLE
Another round of documentation updates

### PR DESCRIPTION
**Pull request by DanVanAtta**
I took another pass at cleaning up the docs. Wanted to combine files a bit and fix up the standards doc. I did a pretty decent edit of the standards file, and re-balanced the content a bit between 'new dev setup', 'code format', and the 'standards' file. There was a 'tips' file that was merged wholesale with the new dev setup.

### CODE REVIEW

File: docs/dev/code_standards.md [hunk 1]
```diff
@@ -1,180 +1,151 @@
...
+Example:
+
+```
+int first = 2;
+int firstSquared = first * first;
+
+int second = 3;
+int secondSquared = second * second;
+
+double distance = Math.sqrt(firstSquared + secondSquared);</code></pre>
```
Comment (ssoloff) [inline docs/dev/code_standards.md]: Looks like some dangling HTML close tags got copy-pasted here.

File: docs/dev/code_standards.md [hunk 2]
```diff
@@ -1,180 +1,151 @@
...
+
+Instead of:
+```
+int first = 2;
+int second = 3;
+
+int firstSquared = first * first;
+int secondSquared = second * second;
+
+double distance = Math.sqrt(firstSquared + secondSquared);</code></pre>
```
Comment (ssoloff) [inline docs/dev/code_standards.md]: Looks like some dangling HTML close tags got copy-pasted here.

**Comment by DanVanAtta** [comment]
@ssoloff updated

**Comment by ron-murhammer** [comment]
Looks good to me. Merging.

**Pull request merged**